deployment/bare_metal/.env.dist

```diff
@@ -26,10 +26,11 @@ COMMUNITY_REDEEM_CONTRIBUTION_URL=https://stage1.gradido.net/redeem/CL-{code}
 COMMUNITY_DESCRIPTION="Gradido Development Stage1 Test Community"
 
 # backend
-BACKEND_CONFIG_VERSION=v9.2022-07-07
+BACKEND_CONFIG_VERSION=v10.2022-09-20
 
 JWT_EXPIRES_IN=10m
 GDT_API_URL=https://gdt.gradido.net
+ENV_NAME=stage1
```

Comment:
Too specific for gradido community. This is a handicap for new communities

Reply:
deleted